components/dialog/dialog.html

```diff
@@ -28,19 +54,47 @@
 
     #body {
         padding: var(--padding);
+        min-height: 3.5rem;
```

Comment:
do we need min height?

Reply:
It doesnt look nice with no content height
![image](https://user-images.githubusercontent.com/39292462/220764519-7f6ffbee-2433-4c72-b618-e198296320f0.png)